### PR TITLE
Tabs: expose tab_close_display_policy property

### LINKED ISSUE #9768: Tabs set_tab_close_display_policy
How can I use tab_close with GDScript?

I exposed it and set to `CLOSE_BUTTON_SHOW_ALWAYS` but still no close_button is showing.

Master branch

### PR DESCRIPTION
**Pull request by ghost**
Fixes #9768

### CODE REVIEW

**Comment by AlexHolly** [comment]
Do I need to change anything else to make it work? 
image

**Comment by Noshyaar** [comment]
I think you misunderstand how Tabs and TabContainer works. (I also did until recently.) Any Control/Container can be a child of TabContainer, it's not limited to Tabs. Its name will be used as tab's name.

Tabs control is a bit different. (and IMO its use case is not as common) You create tabs entirely in scripting using `add_tab(title, icon)`. Also, it doesn't come with built-in Container like TabContainer. To use it, you must connect signals and change/close the tabs manually. In the whole engine, it's only used once (that I know of) in as editting scene tabs.

It doesn't really make sense to use Tabs control as TabContainer child unless you arrange nodes in a specific way. Most of the time you want Container as TabContainer child and add more controls to that.

What you want might be 'close tab' feature in TabContainer which I don't think it's implemented yet. Or, you can use Tabs control which supports close button, but a bit more difficult to work with.

File: scene/gui/tabs.cpp [hunk 1]
```diff
@@ -630,6 +630,7 @@ int Tabs::get_tab_idx_at_point(const Point2 &p_point) const {
 
 void Tabs::set_tab_align(TabAlign p_align) {
 
+	ERR_FAIL_INDEX(p_align, 3);
```
Comment (bojidar-bg) [inline scene/gui/tabs.cpp]: Shouldn't you change this `3` into a (yet non-existent) `ALIGN_MAX`

File: scene/gui/tabs.cpp [hunk 2]
```diff
@@ -764,10 +765,17 @@ Rect2 Tabs::get_tab_rect(int p_tab) const {
 }
 
 void Tabs::set_tab_close_display_policy(CloseButtonDisplayPolicy p_policy) {
+
+	ERR_FAIL_INDEX(p_policy, 3);
```
Comment (bojidar-bg) [inline scene/gui/tabs.cpp]: Same here, but with `CLOSE_BUTTON_SHOW_MAX` or `CLOSE_BUTTON_MAX`

File: scene/gui/tabs.cpp (hunk 1 above)
Comment (Noshyaar) [inline scene/gui/tabs.cpp]: I actually used `*_MAX` earlier, then commit-amend'd back to 3. 😛
https://github.com/godotengine/godot/blob/master/scene/gui/label.cpp#L504
I was afraid that it'd affect performance, but I suppose it make sense to use `*_MAX`.

Comment (bojidar-bg) [inline scene/gui/tabs.cpp]: @Noshyaar It gets optimised at compile-time, so it can't affect performance, at all..

**Pull request merged**